CONTRIBUTING.md

```diff
@@ -24,50 +24,73 @@ python3 -c 'import secrets; print(secrets.token_hex(100))' > .credentials/SECRET
+      The actual names are arbitrary and can be configured in the service settings.
 
-   - Put the organisation name and team names into `.settings.py` so it gets picked up :
+    - Put the organisation name and team names into `.settings.py` so it gets picked up :
 
-     ```python
-     GH_ORGANIZATION = "my-org"
-     GH_COMMITTERS_TEAM = "team1"
-     GH_SECURITY_TEAM = "team2"
-     ```
+      ```python
+      GH_ORGANIZATION = "my-org"
+      GH_COMMITTERS_TEAM = "team1"
+      GH_SECURITY_TEAM = "team2"
+      ```
 
-     <!--
-     TODO: this only works for the local dev environment. staging and prod still need work:
-     https://github.com/Nix-Security-WG/nix-security-tracker/issues/239
-     https://github.com/Nix-Security-WG/nix-security-tracker/issues/240
-     https://github.com/Nix-Security-WG/nix-security-tracker/issues/285
-     -->
+      <!--
+      TODO: this only works for the local dev environment. staging and prod still need work:
+      https://github.com/Nix-Security-WG/nix-security-tracker/issues/239
+      https://github.com/Nix-Security-WG/nix-security-tracker/issues/240
+      https://github.com/Nix-Security-WG/nix-security-tracker/issues/285
+      -->
 
-2. For your GitHub user, in **Developer Settings**, generate a new [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+2.  For your GitHub user, in **Developer Settings**, generate a new [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
 
-   This is not strictly necessary just to run the service, but allows for more API calls and is therefore important for a production deployment.
+    This is not strictly necessary just to run the service, but allows for more API calls and is therefore important for a production deployment.
 
-   - Generate new token
-     - In **Resource owner** select the GitHub organisation associated with the application
-     - In **Repository access** select **Public Repositories (read-only)**
-     - In **Permissions**, set **Members** permissions to **Read-only**
-     - No other permissions are required
-   - Store the value in `.credentials/GH_TOKEN`
+    - Generate new token
+      - In **Resource owner** select the GitHub organisation associated with the application
+      - In **Repository access** select **Public Repositories (read-only)**
+      - In **Permissions**, set **Members** permissions to **Read-only**
+      - No other permissions are required
+    - Store the value in `.credentials/GH_TOKEN`
 
-3. In the GitHub organisation settings, [create an OAuth application](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app):
+3.  In the GitHub organisation settings, [register a GitHub application](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app):
```

Comment:
I only touched this third point with my commit, but `prettier` decided that the previous points had to change formatting also.